### PR TITLE
Update the old sam_merge tool.

### PR DESCRIPTION
**Pull request by bgruening**
*  include stdio
* remove `title` field
* use best-practises
* use `multiple=True` insetead of  `<repeat>`
* increase tool version

This will hopefully address: https://biostar.usegalaxy.org/p/10441/

### CODE REVIEW

File: tools/sam_merge/sam_merge.xml [hunk 1]
```diff
@@ -1,62 +1,56 @@
...
+            VALIDATION_STRINGENCY=LENIENT
+            O="$output1"
+            TMP_DIR=$__new_file_path__
+            #for $input in $inputs
+                I="${input}"
+            #end for
+]]>
+    </command>
+    <inputs>
+        <param name="mergeSD" value="true" type="boolean" label="Merge all component bam file headers into the merged bam file"
```
Comment (nsoranzo) [inline tools/sam_merge/sam_merge.xml]: Is ```value="true"``` necessary?

Comment (jj-umn) [inline tools/sam_merge/sam_merge.xml]: I thought boolean had attributes: truevalue=""  falsevalue=""  checked="true"

**Comment by bgruening** [comment]
Nice catch @nsoranzo and @jj-umn the `value` parameter was a leftover from the very early Galaxy days as it seems. I removed it.

**Comment by nsoranzo** [comment]
+1, thanks Björn!

**Comment by bgruening** [comment]
Thanks @nsoranzo and @jj-umn  for reviewing!

**Pull request merged**